source/portable/BufferManagement/BufferAllocation_2.c

```diff
@@ -393,30 +399,34 @@ NetworkBufferDescriptor_t * pxResizeNetworkBufferWithDescriptor( NetworkBufferDe
 {
     size_t xOriginalLength;
     uint8_t * pucBuffer;
+    size_t xSizeBytes = xNewSizeBytes;
```

Comment:
```suggestion
    size_t uxSizeBytes = xNewSizeBytes;
```

Also the function parameter `xNewSizeBytes` should be `uxNewSizeBytes`.